docs: clarify folder deletion instructions for dbt Cloud Studio IDE (#7673)
fishtown-analytics/docs.getdbt.com: website/docs/faqs/Troubleshooting/gitignore.md

<Lightbox src="/img/docs/dbt-cloud/cloud-ide/restart-ide.png" width="50%" title="Restart the IDE by clicking the three dots on the lower right or click on the Status bar" />

4. Select **Restart <Constant name="cloud_ide" />**.

> 5. Go back to your File Catalog and delete the following files or folders if you have them:

Reviewer: Good call out @abhisheksadhula Just going to bring this a little more in line with our branding and style guide
```suggestion
5. Go back to the **File explorer** in the IDE and delete the following files or folders if you have them:
```
